### DISCUSSION TITLE
How to turn on the Tap Game mode?

### DISCUSSION THREAD

**Discussion by alex-morgun** [manna-harbour/miryoku#205] [Q&A] [answered]
```
      board: '["nice_nano_v2"]'
      shield: '["corne_left","corne_right"]'
      kconfig: '["CONFIG_ZMK_DISPLAY=y\nCONFIG_BT_CTLR_TX_PWR_PLUS_8=y\n"]'
      custom_config: '["#define MIRYOKU_TAP_QWERTY\n"]'
```

How to switch to tap mode?

**Comment by manna-harbour** [comment, marked as answer]
See https://github.com/manna-harbour/miryoku/tree/master/docs/reference#additional-features. I.e. hold any thumb key and double tap the top row ring finger key on the same hand. Reset or power cycle the keyboard to exit.

You can customise the Tap layer with https://github.com/manna-harbour/miryoku_zmk#customisation. If your corne has 6 columns you could map the unused keys to return to Base, add mods, etc.

**Comment by alex-morgun** [comment]
Thanks, a quick double tap works!
